include/zephyr/bluetooth/gatt.h

```diff
@@ -1772,9 +1780,17 @@ struct bt_gatt_read_params {
 			bool variable;
 		} multiple;
 		struct {
-			/** First requested handle number. */
+			/** First requested attribute handle number. 
+			 * 
+			 * @note When attributes are read by UUID, this is typically set to 
+			 * @ref BT_ATT_FIRST_ATTRIBUTE_HANDLE.
```

Comment:
I'm not a fan of the `this is typically set to @ref BT_ATT_FIRST_ATTRIBUTE_HANDLE` text. How do you know if that's typical?

I agree that for the first call this will likely be set to `BT_ATT_FIRST_ATTRIBUTE_HANDLE`, but it shouldn't be set to any subsequent calls. 

E.g. if you want to do a discovery for a specific service, you'd search for the service with `BT_ATT_FIRST_ATTRIBUTE_HANDLE`, but any calls to discover the attributes of that service should not have `BT_ATT_FIRST_ATTRIBUTE_HANDLE` as the start handle. 

Ditto for the end handle, which should be set to the end of the service instead. 

In my experience, `BT_ATT_FIRST_ATTRIBUTE_HANDLE` and `BT_ATT_LAST_ATTRIBUTE_HANDLE` are/should probably only used 1/5th of 1/10th of the time, and thus not very typical